viz/cmd/root.go

```diff
@@ -76,6 +77,7 @@ func NewCmdViz() *cobra.Command {
 	vizCmd.PersistentFlags().StringVar(&impersonate, "as", "", "Username to impersonate for Kubernetes operations")
 	vizCmd.PersistentFlags().StringArrayVar(&impersonateGroup, "as-group", []string{}, "Group to impersonate for Kubernetes operations")
 	vizCmd.PersistentFlags().StringVar(&apiAddr, "api-addr", "", "Override kubeconfig and communicate directly with the control plane at host:port (mostly for testing)")
+	vizCmd.PersistentFlags().StringVar(&vizNamespace, "viz-namespace", "", "Override linkerd-viz namespace auto-detection. It skips most of viz healthchecks (useful on clusters with restricted permissions)")
```

Comment:
I don't think this description is quite accurate.  This flag will just skip detecting the namespace by label.  It won't affect which healthchecks are run.

Reply:
Done